e2e/README.md

```diff
@@ -37,3 +37,90 @@ If you have .env file, you can use CI or TEST_PROVIDER environment variables to
 - `TEST_PROVIDER=sauce npm run test:e2e`
 
 Both command do the same, use the saucelabs as a driver, but you get the results and the evidences locally.
+
+### pictures
+
+The results of the tests are written in the terminal, but you can check the created pictures about the elements in the same folder where the tests are located in the ./pics folder, next to the references.
+
+### run only one test file
+
+If you want to run just one file you can use the --spec argument. This argument not works with `npm run test:e2e`, you have to use `./node_modules/.bin/wdio`. Here is an example to run just the buttuns-test.js:
+
+- `./node_modules/.bin/wdio wdio.conf.js --spec ./e2e/test/buttons-tests/buttons-test.js`
```

Comment:
wrong! `npm run test:e2e -- --spec ./e2e/test/buttons-tests/buttons-test.js` will work. more info about the double dash magc: https://docs.npmjs.com/cli/run-script